contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -615,21 +615,25 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
     }
 
     /**
-     * Calculates the total notional rebalance size. This can be used by external contracts and keeper bots to calculate the optimal exchange to rebalance with.
+     * Calculates the chunk rebalance size. This can be used by external contracts and keeper bots to calculate the optimal exchange to rebalance with.
      * Note: this function does not take into account timestamps, so it may return a nonzero value even when shouldRebalance would return ShouldRebalance.NONE for 
      * all exchanges (since minimum delays have not elapsed)
      *
-     * @return uint256      Total notional rebalance size. Measured in the asset that would be sold
-     * @return bool         Whether it would be levering or delevering
+     * @param _exchangeNames    Array of exchange names to get rebalance sizes for
+     *
+     * @return sizes            Array of total notional rebalance size. Measured in the asset that would be sold
```

Comment:
`Array of total notional chunk size`

Reply:
Think the amounts should always be the same asset, the collateral asset.

Reply:
Open to reasons that it shouldn't be though

Reply:
We have it as the amount of the asset being sold. Since we are performing exact input trades, this is the relevant metric for keeper bots to calculate the optimal exchange.